src/khoj/routers/api_chat.py

```diff
@@ -73,6 +73,48 @@
 from khoj.routers.email import send_query_feedback
 
 
+class FilterRequest(BaseModel):
```

Comment:
Good call to add a request model! Can you add this in `khoj.utils.rawconfig` and import from there?